core/templates/dev/head/css/oppia.css

```diff
@@ -452,6 +452,7 @@ textarea {
   opacity: 0.5;
   padding: 4px 8px;
   position: fixed;
+  z-index: 3;
```

Comment:
Could you please add a comment here saying:

/* This is needed so that the exploration summary tiles don't overlap the button in the library page. */

Thanks! (Also, don't forget to reply to this comment when you've done it -- see instructions at step 5 [here](https://github.com/oppia/oppia/blob/develop/CONTRIBUTING.md#instructions-for-making-a-code-change).)